pkg-manager/core/src/install/link.ts

```diff
@@ -134,6 +135,53 @@ export async function linkPackages (
     failOnMissingDependencies: true,
     skipped: new Set(),
   })
+  let linkedToRoot = 0
```

Comment:
This section is just moving the `sym link` logic up before the `linkNewPackages` (which invokes the fetcher internally)

It is otherwise copy-pasted.

Now the directory fetcher will have locally linked packages setup as necsessary